instruqt/avoiding-installation-pitfalls/01-working-with-preflight-checks/assignment.md

```diff
@@ -125,18 +130,18 @@ Getting Started
 Let's start by creating a new file for our preflight checks
```

Comment:
Might just mention the full path you expect to see the preflight file - I think in the screenshot you can tell it should go in the replicant's release folder, but without expanding it the screenshot is very small

Reply:
Ah actually, on re-reading, I can see that you want it to go in the home folder - so I guess my note would be, at this point, I wasn't sure where you expected the file to be created - like I can see that it should be in the working directory "./slackernews-preflight" but I wasn't sure if I missed a step to cd somewhere or not, but expanding the screenshot helps

Reply:
Added some text to address this.